src/theme/components/customTable.ts

```diff
@@ -0,0 +1,29 @@
+const baseStyle = ({ showRowDividers, columnHeadersTextColor }) => ({
+  th: {
+    border: showRowDividers ? undefined : "none",
+    borderBottom: showRowDividers ? undefined : "0px",
+    fontWeight: "bold",
+    textTransform: "capitalize",
+    fontSize: "sm",
```

Comment:
@bigfishdesign13 what is the theme equivalent for "sm" here?

Reply:
@EdwinGuzman right now Marty told me use fontSize: "0". In future syntax is going to change